cmd/sprint-automation/main.go

```diff
@@ -245,6 +245,9 @@ func users(client *pagerduty.Client, slackClient *slack.Client) (map[string]stri
 func usersOnCallAtTime(client *pagerduty.Client, slackClient *slack.Client, when time.Time) (map[string]string, []error) {
 	var errors []error
 	userIdsByRole := map[string]string{}
+	// 7 am UTC is when our PD day begins, and US on-call ends at 10pm UTC. Query 8 am - 9 pm for safe results
```

Comment:
> US on-call ends at 10pm UTC.

Is this the calculation?
18:00 (EDT) = 22:00 (UTC)

Do we need to modify the query again if we have a teammate from PDT?
I guess not since PDT starts work as well at range of the current query.

Reply:
Yeah, I just got that value from looking at the L3 support schedule on PD. EU starts at 7 am UTC, and US ends at 10pm UTC (so 6pm eastern). Even when we had team members on the west coast those hours were the same, but if that were to change in the future the query should still work as is. The point of it is just to make sure that we hit at least a second in all of the possible blocks (currently EU and US) to pick everything up.